File: TypeScript/URL/get-port-from-url.ts
import * as url from "url"

function main(): void {
    // Get the port from a given url.
    console.log(getPortFromURL("https://www.example.com:8080"));
}

main()

// Get the port from a given url.
function getPortFromURL(givenURL: string): string | null {
    return new URL(givenURL).port;
}

File: Python/URL/get_port_from_url.py
import urllib.parse


# Get the port from a given url.
def get_port_from_url(url):
    return urllib.parse.urlparse(url).port


def main():
    # Get the port from a given url.
    print(get_port_from_url("https://www.example.com:80"))


main()